CHANGELOG.md

```diff
@@ -8,12 +8,21 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/).
 
 [//]: # "Add stuff here"
 
-## [1.3.4]
+## [1.3.5]
 
 ### Changed
 
-- AI instructions improvement
-- Activated NPL-dev for all workspaces containing NPL files
```

Comment:
This was already documented earlier in the changelog and not part of this release